tests/test_crs/valid_hammerdb.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
+kind: Benchmark
+metadata:
+  name: hammerdb
+  namespace: my-ripsaw
+spec:
+  workload:
+    name: "hammerdb"
+    args:
+      db_init: false
+      db_benchmark: true
+      db_server: "172.30.113.106"
```

Comment:
we'll either need to edit this in the test script or replace with the fqdn of the mssql pod. take a look at ycsb test

Reply:
changed to mssql-deployment.sql-server